app/javascript/components/layout/Header.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Navbar, Nav, Form } from 'react-bootstrap';
+
+class Header extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <Navbar bg="primary" variant="dark" expand="lg" className="mb-3">
+        <Navbar.Brand href={this.props.report_mode ? '/' : this.props.root}>
+          Sara Alert<small className="nav-version ml-1">{this.props.version}</small>
+        </Navbar.Brand>
+        {this.props.current_user && (
+          <React.Fragment>
+            <Nav className="pt-1 mr-auto" activeKey={window.location.pathname}>
+              {this.props.current_user.can_see_enroller_dashboard_tab && (
+                <Nav.Link className="ml-3" href={`${window.BASE_PATH}/patients`}>
+                  <i className="fas fa-table"></i>&nbsp;&nbsp;Enroller Dashboard
```

Comment:
Instead of using `&nbsp;`, it's probably better adding a margin using the built in bs classes to the icon (maybe `mr-1`)

Reply:
I would also recommend making all of these font awesome icons fixed width (add `fa-fw` class)

Reply:
Good comments, thanks. Done and done.